src/Profile.php

```diff
@@ -1597,6 +1603,60 @@ public function showFormTracking($openform = true, $closeform = true)
         $matrix_options['title'] = _n('Followup', 'Followups', Session::getPluralNumber()) . " / " . _n('Task', 'Tasks', Session::getPluralNumber());
         $this->displayRightsChoiceMatrix(self::getRightsForForm('central', 'tracking', 'followups_tasks'), $matrix_options);
 
+        echo "<div class='mt-n2 mx-n2 mb-4'>";
+        echo "<table class='table table-hover card-table'>";
+        echo "<thead>";
+        echo "<tr><th colspan='2'><h4>" . __s('Users mentions') . "<h4></th></tr>";
+
+        echo "</thead>";
+
+        echo "<tbody>";
+
+        $description = __s('Enables or disables the ability to mention users within the application.') . "<br><br>";
+        $description .= "<b>" . __s('Disabled') . "</b> : " . __('User mentions are disabled for this profile.') . "<br><br>";
+        $description .= "<b>" . __s('Full') . "</b> : " . __('Displays all users.') . "<br><br>";
```

Comment:
Perhaps specify that the user will be added as an observer if they are not already a ticket actor?